astro/src/content/docs/_shared/_collected-metrics.mdx

```diff
@@ -106,11 +85,11 @@ The FusionAuth [license agreement](/license) covers the specifics of how we coll
 
 We know how critical compliance is for your business. For many customers, it's a primary driver in why they chose FusionAuth. Because of this, we made the choice to avoid collecting any personally identifiable information. We collect all stats in aggregate, so that there is no chance of violating confidentiality or compliance.
 
-### What Steps are You Taking to Avoid Scope Creep and Collecting Ever-More Data?
+### What Steps Are You Taking to Avoid Scope Creep and Collecting Ever-More Data?
 
 We're aware of how easy it is to let a project like this expand. Who doesn't like more data? Before we started this project, we decided on the following principles:
 
-- Self-hosting customers will always be able to turn off usage data collection.
+- Self-hosted customers will always be able to turn off usage data collection.
```

Comment:
Probably should add self-hosted vs self-hosting to the devdocs readme for consistency. No preference on which term to use.